2023/did-wg.html

```diff
@@ -180,6 +180,7 @@ <h2>Scope</h2>
 
         <p>
 The Working Group may also publish new Working Group Notes.
+Notes should not include submissions for DID Methods and should not address issues regarding any particular DID Method. The WG welcomes DID Method standardization outside the WG (whether in the W3C or in other standards bodies).
```

Comment:
```suggestion
Notes should not include submissions for DID Methods and should not address issues regarding any particular DID Method. The WG welcomes DID Method standardization outside the WG (whether in another W3C or in other standards bodies).
```

(I presume that is the intention...)